src/routers/index.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 import { Router, Request, Response } from 'express';
```

Comment:
remove those unused vars

Reply:
you meant i need to remove those  eslint comments?